code/frameworks/nextjs/README.md

```diff
@@ -0,0 +1,622 @@
+# Storybook for Next.js <!-- omit in toc -->
```

Comment:
This README's content was mostly copied (with minor adaptations) from https://github.com/RyanClementsHax/storybook-addon-next/blob/main/README.md